content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+that column, click anywhere on the thin gray column that says the stage name and the column expands,
+revealing the applicants.
+
+Each stage has a color-coded bar beneath the stage name, giving information about the applicant's in
+the stage. The colors are:
+
+- :guilabel:`Green`: There are applicants in the stage with an activity that is either scheduled or
+  needs to be scheduled in the future.
+- :guilabel:`Yellow`: There are applicants in the stage with an activity either scheduled for today
+  or due to be scheduled today.
+- :guilabel:`Red`: There are applicants in the stage with overdue activities which need to be
+  scheduled.
+- :guilabel:`Gray`: There are either no activities currently scheduled and no future activities that
+  are required to be scheduled, or there are no applicants currently in the stage.
+
+.. image:: recruitment/stages.png
+   :align: center
+   :alt: Expand a folded column by clicking on it in the kanban view.
+
+Customize stages
+================
+
+Stages can be modified, added, or deleted to best meet the needs of the hiring flow of the business.
+
+New stage
+---------
+
+To create a new stage, click on :guilabel:`+ Add a Column` and a new column appears. Enter the title
+for the new stage in the :guilabel:`Column title` field, then click :guilabel:`Add`. The new column
+appears, and another new stage is available to create. If no new stages are needed, click anywhere
+on the screen to close the new column.
+
+.. image:: recruitment/add-column.png
+   :align: center
+   :alt: Add a column by clicking on Add Column.
+
+.. _modify:
+
+Modify stage
+------------
+
+To modify the settings of a stage, hover over the name of the stage, and a :guilabel:`⚙️ (gear)`
+icon appears. Click on the :guilabel:`⚙️ (gear)` icon and a menu appears. Click on the
+:guilabel:`Edit Stage` option. An :guilabel:`Edit Column` form appears. Make any modifications to
+the form, then click :guilabel:`Save` when done.
+
+Edit column form
+~~~~~~~~~~~~~~~~
+
+The :guilabel:`Edit Column` form is where the stage's settings are configured. The only required
+fields are the :guilabel:`Stage Name`, and the :guilabel:`Tooltips` section.
+
+The fields to be populated or modified are:
+
+- :guilabel:`Stage Name`: Type in a name for the stage.
+- :guilabel:`Email Template`: Select an email template to be used from the drop-down menu. If a
+  template is selected, when the applicant card enters the stage, an email will automatically be
+  sent to the applicant using the selected template.
+- :guilabel:`Folded in Kanban`: Check the box to have the stage appear folded (hidden) at all times
+  in the default view.
+- :guilabel:`Hired Stage`: Check the box if this stage indicates that the applicant is hired. When
+  an applicant's card enters this stage, the card displays a :guilabel:`Hired` banner in the corner.
+  If this box is checked, this stage is used to determine the hire date of an applicant.
```

Comment:
Yup= Jess and I had to tinker a lot to find that out (I'll be honest, it was 99% Jess!) because you can set MULTIPLE STAGES as the hired stage! Which is crazy- and can't be fixed because the kanban is linked to other apps, so making a rule for that is not possible according to the devs (too many potential problems). So, we learned that the first stage that is set as a hiring stage is the date the applicant is hired (on their employee record).  After hashing it out with Jess, I decided not to mention that you can set multiple stages as the hired stage because that's crazy, unhelpful, and will likely confuse people (since instinctively, there should only be one hired stage). Plus if I mentioned it, I think it makes Odoo look bad, and thought best not to include that.